M2Mqtt/Messages/MqttMsgBase.cs

```diff
@@ -211,6 +214,20 @@ object GetStringObject(object value)
                 return sb.ToString();
             }
 
+            MqttQoSLevel[] qosLevel = value as MqttQoSLevel[];
+            if (qosLevel != null)
+            {
+                string hexChars = "0123456789ABCDEF";
```

Comment:
Why not use the tostring from mscorlib?

Reply:
I basically just reuse what was there. Let me adjust.